src/main/java/com/amazon/opendistroforelasticsearch/knn/common/KNNConstants.java

```diff
@@ -21,12 +21,33 @@
     public static final String HNSW_ALGO_EF_CONSTRUCTION = "efConstruction";
     public static final String HNSW_ALGO_EF_SEARCH = "efSearch";
     public static final String HNSW_ALGO_INDEX_THREAD_QTY = "indexThreadQty";
-    public static final String L2 = "l2";
-    public static final String L1 = "l1";
-    public static final String LINF = "linf";
-    public static final String COSINESIMIL = "cosinesimil";
-    public static final String HAMMING_BIT = "hammingbit";
-    public static final String INNER_PROD = "innerproduct";
     public static final String DIMENSION = "dimension";
-    public static final String KNNEngine = "knnEngine";
+    public static final String KNN_ENGINE = "engine";
```

Comment:
for future: lets sort constants based on alphabets. This will help readability and easy to search specific constant.

Reply:
Agree. Do you think that it makes sense to localize constants to specific files or keeping all constants in a single class?

Reply:
I prefer single file, it is easy to maintain.

Reply:
Makes sense. We have some what of a hybrid approach right now. But we can adjust in the future